src/controller/backend/Endpoints/IdentityEndpoints.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using System.Text;
+using ApiSchema.Identity;
+using Data.System;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.IdentityModel.Tokens;
+
+namespace Backend.Endpoints;
+
+public static class IdentityEndpoints
+{
+    public static void MapIdentityEndpoints(this IEndpointRouteBuilder endpoints, RsaSecurityKey key)
+    {
+        var group = endpoints.MapGroup("/api/identity");
+
+        group.MapPost("/login", async (
+            [FromBody] LoginModel model,
+            [FromServices] UserManager<SystemUser> userManager,
+            HttpContext context) =>
+        {
+            const string errorMessage = "Invalid username or password";
+            
+            var user = await userManager.FindByNameAsync(model.UserName);
+            
+            if (user is null)
+                return Results.BadRequest(errorMessage);
+            
+            if (!await userManager.CheckPasswordAsync(user, model.Password))
+                return Results.BadRequest(errorMessage);
+            
+            var handler = new JwtSecurityTokenHandler();
+            var token = handler.CreateToken(new SecurityTokenDescriptor
+            {
+                Subject = new ClaimsIdentity([
+                    new Claim(ClaimTypes.NameIdentifier, user.Id),
+                    new Claim(ClaimTypes.Name, user.UserName!),
+                    //new Claim("role", user)
```

Comment:
@NotSayo have you checked if it would make sense to also include other claims and roles in the token other than the IF identifier and the username?

Reply:
I will think about how to implement this one